src/index.d.ts

```diff
@@ -1305,6 +1305,76 @@ export interface SearchBarBase extends TextInputProperties {
   onChangeText?(text: string): void;
 }
 
+export class Tooltip extends React.Component<TooltipProps, any> {}
+
+export interface TooltipProps {
+  /**
+   * sets backgroundColor of the tooltip and pointer.
+   */
+  backgroundColor?: string;
+  /**
+   * Color to highlight the item the tooltip is surrounding.
+   */
+
+  highlightColor?: string;
+  /**
+   * function which gets called on closing the tooltip.
+   */
+
+  onClose?: () => any;
+  /**
+   * function which gets called on closing the tooltip.
+   */
+
+  onOpen?: () => any;
+  /**
+   * function which gets called on opening the tooltip.
+   */
+
+  pointerColor?: string;
+  /**
+   * Color of tooltip pointer, it defaults to the backgroundColor if none passed .
+   */
+
+  toggleOnPress?: Function;
```

Comment:
For functions the type:
`functionName(param: paramType): return value`

In our case:
`toggleOnPress?(): void;`